tests/test_prompt_actions.py

```
@@ -24,33 +28,47 @@ def test_tool_guide_documents_native_tool_protocol_first() -> None:
+
 def test_tool_guide_uses_schema_derived_aliases_and_required_fields() -> None:
     guide = get_tool_guide()
     schemas = get_edit_tool_schema_map()
 
     assert "Required fields: `path`, `content`" in guide
-    assert "Required fields: `path`, `operation`" in guide
-    assert "Operation types: " in guide
+    assert "Required fields: `path`, `old_string`, `new_string`" in guide
+    assert "Required fields: `patch`" in guide
 
     for tool_name, schema in schemas.items():
```

Ruff (0.15.12)

[warning] Loop control variable `tool_name` not used within loop body

Rename unused `tool_name` to `_tool_name`

(B007)